modules/installation-osp-accessing-api-no-floating.adoc

```diff
@@ -3,10 +3,88 @@
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-kuryr"]
+:osp-kuryr:
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user"]
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user-kuryr"]
+:osp-kuryr:
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-restricted"]
+:osp-ipi:
+:osp-restricted:
+endif::[]
 
 [id="installation-osp-accessing-api-no-floating_{context}"]
-= Enabling access without floating IP addresses
+= Completing installation without floating IP addresses
+
+You can install {product-title} on {rh-openstack-first} without using a floating IP address and external network. In the
+ifdef::osp-ipi[`install-config.yaml`]
+ifdef::osp-upi[`inventory.yaml`]
+file, leave the following
+ifdef::osp-ipi[parameters]
+ifdef::osp-upi[variables]
+blank:
+
+ifdef::osp-ipi[]
+* `platform.openstack.ingressFloatingIP`
+* `platform.openstack.lbFloatingIP`
+* `platform.openstack.externalNetwork`
```

Comment:
leave the externalNetwork will make the generated subnet have no gateway and can not reach outside, this make bootstrap no way to fetch ignition file for itself from OSP glance api.

Reply:
@wjiangjay Is there a BZ ticket that discusses this issue more? I was told explicitly that externalNetwork could remain blank in for an IPI w/out FIPs scenario.

Reply:
^^^ @iamemilio @mandre

Reply:
@maxwelldb it's not an issue when without externalNetwork, it just need some additional steps to make the whole things work as our expected.